utils/emoji_utils.jsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export function sortEmojis(emojiA, emojiB, searchedName) {
+    const aName = emojiA.name || emojiA.aliases[0];
+    const bName = emojiB.name || emojiB.aliases[0];
```

Comment:
Do we really only need to consider the first alias?

Reply:
I think it makes sense to just check the first alias since that matches what the user would see as suggested text. If we happen to find a weird case later on, we can always change it or add a special case.